spec/spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+ENV['RACK_ENV'] = 'test'
+require 'minitest/autorun'
+require 'rack/test'
+require 'nokogiri'
+
+Dir.glob('./app/{controllers}/*.rb').each { |file| require file }
+
+include Rack::Test::Methods
+
+DB.drop_table?(:users)
```

Comment:
Instead of dropping and creating the users table here, we should just run the rake tasks for migrations. Running `RACK_ENV=test rake db:migrate` should run the migrations for our test db. 

We want to avoid creating/dropping tables every time we run our test suite because this will slow down when our test suite becomes huge. Also, this may introduce a difference between the schema in tests and in development/production. Using the same migrations for all our environments will remove that possibility.

Let's use transactional tests http://sequel.jeremyevans.net/rdoc/files/doc/testing_rdoc.html. This will ensure that all interactions with the DB will only last one test case. Every test case will start on a clean DB.